src/ChartTemplates/StackChart/StackChartTemplate.js

```diff
@@ -30,7 +30,7 @@ export const StackChart = ({ ...props }) => {
     const rawData = props.data.length && props.data.filter(item => item.y > 0).map(el => el.y);
     const dataMin = rawData.length && rawData.reduce((acc, curr) => Math.min(acc, curr));
 
-    const LegendLabel = ({ ...rest }) => <a className="pf-c-button pf-m-link pf-m-inline"><ChartLabel { ...rest }/></a>;
+    const LegendLabel = ({ ...rest }) => <a href={ props.legendClick } className="pf-c-button pf-m-link pf-m-inline"><ChartLabel { ...rest } /></a>;
```

Comment:
@christiemolloy  EEEE SORRY I MISSED THIS (kinda surprised no one yelled at us for the regression)